ldap-client.go

```diff
@@ -41,7 +45,10 @@ func (lc *LDAPClient) Connect() error {
 				return err
 			}
 		} else {
-			l, err = ldap.DialTLS("tcp", address, &tls.Config{InsecureSkipVerify: false})
+			l, err = ldap.DialTLS("tcp", address, &tls.Config{
+				InsecureSkipVerify: lc.InsecureSkipVerify,
```

Comment:
For additional detail, this is the upstream commit that requires this change:

https://github.com/golang/go/commit/fca335e91a915b6aae536936a7694c4a2a007a60